assets/src/blocks/Media/setupMediaElementJS.js

```diff
@@ -1,27 +1,29 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
 import { addScriptTag } from '../../components/useScript/addScriptTag';
 import { addLinkTag } from '../../components/useStyleSheet/addLinkTag';
 
 export const setupMediaElementJS = function() {
   const meJSNodes = document.querySelectorAll( '.mejs-video-block' );
 
-  if (meJSNodes.length === 0) {
+  if ( meJSNodes.length === 0 ) {
     return;
   }
 
-  addLinkTag({ href: 'https://cdn.jsdelivr.net/npm/mediaelement@4.2.16/build/mediaelementplayer-legacy.min.css' });
-  
-	const onLoad = () => {
-    meJSNodes.forEach(node => {
-      const player = new MediaElementPlayer(node, {
+  addLinkTag( { href: 'https://cdn.jsdelivr.net/npm/mediaelement@4.2.16/build/mediaelementplayer-legacy.min.css' } );
+
+  const onLoad = () => {
+    meJSNodes.forEach( ( node ) => {
+      const player = new MediaElementPlayer( node, {
```

Comment:
Since both `/* eslint-disable no-undef */` and `/* eslint-disable no-unused-vars */` apply only to this line, you could write them here inline

Reply:
Done.